src/components/BRB/BRBParallax.tsx

```diff
@@ -141,6 +141,7 @@ function BRBParallax() {
 
 const Container = styled.div`
   width: 100%;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family